src/RA_Dlg_RichPresence.h

```diff
@@ -0,0 +1,41 @@
+#ifndef RA_DLG_RICHPRESENCE_H
+#define RA_DLG_RICHPRESENCE_H
+#pragma once
```

Comment:
This file should no longer exist. I'm guessing it got readded to resolve a conflict.

Reply:
I'll get rid of it